code/modules/mob/living/brain/MMI.dm

```diff
@@ -190,6 +193,7 @@
 
 /obj/item/mmi/examine(mob/user)
 	..()
+	to_chat(user, "<span class='notice'>There is a switch to toggle the radio system [radio.on ? "off" : "on"].[brain ? " that is currently being covered by the brain." : ""]</span>")
```

Comment:
There is a switch to toggle the radio system on. that is currently being covered by the brain.

Reply:
oof my b

Reply:
same